packages/presentation/lib/src/pages/login_page/bloc/login_data.dart

```diff
@@ -1,28 +1,40 @@
+import 'package:domain/domain.dart';
+
 class LoginData {
-  final String errorMessage;
   final String login;
   final String password;
+  final String? loginValidation;
+  final String? passwordValidation;
 
   const LoginData({
-    required this.errorMessage,
     required this.login,
     required this.password,
+    this.loginValidation,
+    this.passwordValidation,
   });
 
   LoginData copyWith({
-    String? errorMessage,
     String? login,
     String? password,
-  }) {
-    return LoginData(
-      errorMessage: errorMessage ?? this.errorMessage,
-      login: login ?? this.login,
-      password: password ?? this.password,
-    );
-  }
+    String? loginValidation,
+    String? passwordValidation,
+  }) =>
+      LoginData(
+        login: login ?? this.login,
+        password: password ?? this.password,
+        loginValidation: loginValidation,
+        passwordValidation: passwordValidation,
+      );
+
+  LoginData fromValidationResult({required ValidationResult result}) =>
```

Comment:
move it into mapper

Reply:
decided to delete and use .copyWith instead